src/lib/services/instance-stats-service.ts

```diff
@@ -207,6 +243,19 @@ export class InstanceStatsService {
         ];
     }
 
+    async getAppCountSnapshot(range: string): Promise<number> {
+        return this.snapshot
+            ?.then((stats) => {
+                return (
+                    stats?.clientApps?.find((t) => t.range === range)?.count ||
```

Comment:
`t` throws me off here. If it's a clientApp, why not use `clientApp` as the parameter name?

Reply:
Yeah, naming naming naming :P it's `t` for tuple, but I think `clientApp` representing (range, count) pair is more meaningful